test/tests.js

```diff
@@ -523,6 +550,32 @@ describe('Class Locker', function() {
 					throw err;
 				});
 		});
+
+		it('should send write lock heartbeats', function() {
+			this.timeout(10000);
+			let rwlock;
+			return locker.writeLock('key', { maxWaitTime: 1, lockTimeout: 1 })
+				.then((_rwlock) => {
+					rwlock = _rwlock;
+					return pasync.setTimeout(2000);
+				})
+				.then(() => {
+					let otherLocker = new Locker(redizClient);
+					return otherLocker.writeLock('key', { maxWaitTime: 1 })
+						.then(() => {
+							throw new Error('Expected lock to fail');
+						}, () => {
+							// Expected error, do not throw
```

Comment:
any chance we can get some expect statements here? Leads to less confusion on what you are actually testing.